MCprep_addon/materials/generate.py

```diff
@@ -367,6 +394,10 @@ def set_texture_pack(
 	run the swap (and auto load e.g. normals and specs if avail.)
 	"""
 	mc_name, _ = get_mc_canonical_name(material.name)
+	texture_pack_format = get_format_version_texturepack(folder)
+	# grass rename to short_grass in MC 1.20.3 with pack format version 22
+	if (material.name == "grass" and texture_pack_format > 21):
+		mc_name = "short_grass"
```

Comment:
Maybe we should extend this further to all changes, just to properly handle older resource packs, though I'll defer that decision to @TheDuckCow

Also, I think we should make sure that the OBJ actually needs this particular change. OBJs from worlds predating this change won't need us to handle pack format 22 compatibility, though checking might be a bit of a pain (this is making me wish CommonMCOBJ V1 included the world's version in the header, probably will open a proposal for that for V2 in the next couple of days)